src/BaseAWSConnector.ts

```diff
@@ -169,7 +183,9 @@ class AWSIdentity {
           await iam
             .putRolePolicy({
               PolicyDocument: JSON.stringify(policy),
-              PolicyName: `policy_${roleName}_${crypto.randomBytes(4).toString('hex')}`,
+              PolicyName: `policy_${roleName}_${crypto
+                .randomBytes(4)
+                .toString('hex')}`,
```

Comment:
IMO the change here makes the line less readable :(

Reply:
Fixed by col width